test/e2e/apimachinery/custom_resource_definition.go

```diff
@@ -251,6 +256,119 @@ var _ = SIGDescribe("CustomResourceDefinition resources [Privileged:ClusterAdmin
 		}
 	})
 
+	/*
+		Release : v1.17
+		Testname: Custom Resource Definition, defaulting
+		Description: Create a custom resource definition without default. Create CR. Add default and read CR until
+		the default is applied. Create another CR. Remove default, add default for another field and read CR until
+		new field is defaulted, but old default stays.
+	*/
+	framework.ConformanceIt("custom resource defaulting for requests and from storage works ", func() {
```

Comment:
start this as a regular non-conformance e2e, then promote before freeze once we demonstrate it is not flaky

Reply:
Conformance follow-up https://github.com/kubernetes/kubernetes/pull/84861